wormchain/app/app.go

```diff
@@ -180,6 +195,13 @@ var (
 		wormholemoduletypes.ModuleName: nil,
 		// this line is used by starport scaffolding # stargate/app/maccPerms
 		wasm.ModuleName: {authtypes.Burner},
+		tokenfactorytypes.ModuleName: {authtypes.Minter, authtypes.Burner},
+	}
+
+	tokenFactoryCapabilities = []string{
+		tokenfactorytypes.EnableBurnFrom,
+		tokenfactorytypes.EnableForceTransfer,
+		tokenfactorytypes.EnableSetMetadata,
```

Comment:
Add comment on why we need this capabilities?

Reply:
I can remove them as the contract does not require those features. Their associated test cases will need to be commented out too for the test suite to pass. This also surfaced an upstream bug in IsCapabilityEnabled() when the list is empty which I'll update and eventually upstream too.